apps/src/aichat/constants.ts

```diff
@@ -3,7 +3,9 @@ import {
   ChatCompletionMessage,
   Role,
   AITutorInteractionStatus as Status,
+  ModelDescription,
 } from './types';
+import modelsJson from '../../static/aichat/modelDescriptions.json';
```

Comment:
minor: you can replace this path with `@cdo/static/...` since we have a mapping for that in our tsconfig